.svnignore

```diff
@@ -25,3 +25,4 @@ composer.lock
 package.json
 to-test.md
 yarn.lock
+.github
```

Comment:
`.svnignore` file in a git repository. @jeherve - can we remove it?

Reply:
That file is currently used when pushing a release from here to svn:
https://github.com/Automattic/vaultpress/blob/master/bin/release.php#L100

I would suggest removing it, but not in this PR. It should be part of another refactor to update the way W.org SVN releases happen from this repo.

That's most likely a discussion worth having on a P2, but it may be worth moving this repo into the Jetpack monorepo, alongside [the other plugins there](https://github.com/Automattic/jetpack/tree/master/projects/plugins), so we could benefit from all the tooling that's already there. That would allow us to get rid of `.svnignore`, among other things.

Reply:
Sure, makes sense. Thanks for providing a context here!